src/calculator_ROC_class.R

```diff
@@ -7,35 +7,35 @@ ROC <- R6Class("ROC",
     initialize = function() {
     },
     
-    getBestThresholdAndError = function(datos){
+    getBestThresholdAndError = function(datos) {
         self$rOC <- private$calculateROC(datos)
         self$rOC <- private$addCriterion(self$rOC)
         mejoresUmbrales <- self$rOC %>% 
             group_by(criterio) %>%
             summarize(umbral = median(umbrales),
```

Comment:
Creo que esto se verá mejor así:
```suggestion
            summarize(
                umbral = median(umbrales),
                error = median(pError)
            ) %>%
```